frontend/language/src/nb.json

```
@@ -662,6 +662,7 @@
   "process_editor.unknown_heading_error_message": "Obs, noe gikk galt!",
   "process_editor.unknown_paragraph_error_message": "En feil oppstod ved innlasting av BPMN-prosessen. Undersøk at filen er på et gyldig BPMN-format.",
   "process_editor.unsaved_changes": "Du har {{ count }} ulagrede endringer",
+  "process_editor.unsaved_changes_confirmation_message": "Du har ulagrede endringer. Er du sikker på at du vil forlate siden uten å lagre?",
```

Is this message a little shorter and easier to read?
```suggestion
  "process_editor.unsaved_changes_confirmation_message": "Du har ulagrede endringer. Vil du forlate siden uten å lagre?",
```